server/job.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"github.com/google/uuid"
+	pb "github.com/romanlevin/felek/jobs"
+	"log"
+	"os/exec"
+	"sync"
+)
+
+type job struct {
+	id        string    // A UUID string
+	cmd       *exec.Cmd // The underlying Cmd object wrapping the job process
+	stopped   bool      // Has this job been stopped by a client?
+	exitError error     // The error returned by Wait on the Cmd
+	owner     string    // The username of the user who started the job
+	lock      *sync.RWMutex
```

Comment:
Mutex doesn't need to be a pointer here, as long as your functions have pointer receivers.